CMakeLists.txt

```
@@ -248,7 +251,8 @@ write_basic_package_version_file(
 # generate a CMake configuration file for consumption by CMake's `find_package`
 # intrinsic
 #
-install( TARGETS stlab coroutines EXPORT stlabTargets )
+
+install( TARGETS stlab EXPORT stlabTargets )
```

Wouldn't we want to cover the coroutines == true case? What am I missing? 

```suggestion
if( stlab.coroutines )
    install( TARGETS stlab coroutines EXPORT stlabTargets )
else()
    install( TARGETS stlab EXPORT stlabTargets )
endif()
```

---

Applied your change. The way coroutines is handled is very strange. The target is essentially used to change compilation flags of the dependent target. This will all be reworked in future updates here to follow best practice CMake conventions.